### PR TITLE
Use docker container for docker compose file

### PR DESCRIPTION
**Pull request by dbkr**
Rather than copying the source in here

### CODE REVIEW

**Review by robintown — COMMENTED** [review]
This looks fine, would you mind also updating the README instructions though?

**Comment by robintown** [comment]
Also, the lk-jwt-service appears to still be private. Presumably we should publicize it now?

**Review by daniel-abramov — APPROVED** [review]
Thanks for the clean-up!

I think that now it should also address the point from @robintown about the publicity of the jwt service repository.

File: README.md [hunk 1]
```diff
@@ -88,15 +88,12 @@ yarn dev
...
-# Develop backend settings:
-LIVEKIT_KEY="devkey"
-LIVEKIT_SECRET="secret"
-```
-
-Add SFU parameter in your local config `./public/config.yml`:
+To use it, add SFU parameter in your local config `./public/config.yml`:
 
 ```yaml
 "livekit": {
```
Comment (robintown) [inline README.md:99]: Shouldn't this hold just the one `livekit_service_url` option now?

Comment (dbkr) [reply README.md:99]: Ah, good point.

File: README.md [hunk 2]
```diff
@@ -97,7 +97,6 @@ To use it, add SFU parameter in your local config `./public/config.yml`:
...
-    "server_url": "ws://localhost:7880",
     "jwt_service_url": "http:/localhost:8881"
```
Comment (robintown) [inline README.md]:
```suggestion
    "jwt_service_url": "http://localhost:8881"
```

**Review by robintown — APPROVED** [review]

**Pull request merged**